### ISSUE TITLE
Make flutter/flutter new issue template for 1p plugins

### ISSUE THREAD

**Issue by jmagman** [flutter/flutter#117750]
Add instructions for including the versions of all plugins, including their federated versions (maybe `pubspec.lock`?).

https://github.com/flutter/flutter/tree/master/.github/ISSUE_TEMPLATE

**Comment by jmagman** [comment]
Triagers: what additional information would be helpful when looking at flutter/plugin issues?

**Comment by stuartmorgan** [comment]
> Add instructions for including the versions of all plugins, including their federated versions

That's a great idea; this has definitely been an unexpected drawback to federation.

Maybe also specifically ask if it reproduces after a `flutter pub upgrade` as well?

**Comment by exaby73** [comment]
Maybe the inclusion on the OS versions it reproduces on should be considered as well. There have been a few issues I have seen where time could have been saved if the OS version were specified beforehand, though I can't think of any specific issues on the top of my head.

**Comment by jmagman** [comment]
Suggestion from hackers-triage Discord channel: 
> Some of the 1P plugins have had breaking changes recently (webview_flutter for example) so,  a checkbox to confirm from OP that they have read the breaking changes and usage mentioned in readMe and have already made the applicable changes to their code.

**Comment by Hixie** [comment]
There's a new syntax we can use to make the templates into actual forms with separate text fields and so on too, by the way, that didn't exist when we last updated these. Please don't hesitate to make use of that for any of the templates if you think it would improve them.

**Comment by stuartmorgan** [comment]
I used that for the FDE repo template, with a drop-down to pick the plugin that the issue is about (rather than relying on people remembering to say, which they don't always).

**Comment by stuartmorgan** [comment]
Another reason to have the dropdown is that once flutter/plugins is merged into flutter/packages people may not always know what is a plugin and what is just a package. Having an enforced list of options, with instructions saying to use a different template if the thing they are reporting an issue with isn't in that list, should help avoid issues.

**Comment by jmagman** [comment]
Also would be helpful to optionally list what tested platforms _don't_ reproduce the issue.
For example if the issue was reported against iOS, it's useful to know it doesn't reproduce on Android and Windows.

**Comment by huycozy** [comment]
Hi @stuartmorgan, I have just made a PR for this issue. Could you please help to review it?
